src/src/rx_main.cpp

```diff
@@ -715,20 +723,29 @@ void loop()
         }
         else
         {
+            #ifdef NO_SYNC_ON_ARM
+            CURR_RATE_MAX = 4; //switch between 200hz, 100hz, 50hz, rates
```

Comment:
This comment is the same as the original one - one of them must be wrong?

Reply:
I think `CURR_RATE_MAX = 4` breaks 2.4 as it only has 3 air modes.

Reply:
I see Sandro snuck a commit in to fix this https://github.com/AlessandroAU/ExpressLRS/commit/345fe0158d497a75ff5c0252e4f11e932f69e2ff

Reply:
Yup fixed